contracts/dex_grpc/src/contract.rs

```diff
@@ -84,34 +82,31 @@ pub fn execute(
             order_type,
             expiration_time,
             max_amount_out,
-        } => Ok(
-            Response::new().add_message(Into::<CosmosMsg>::into(dex::MsgPlaceLimitOrder {
-                creator: env.contract.address.to_string(),
-                receiver,
-                token_in,
-                token_out,
-                tick_index_in_to_out,
-                limit_sell_price,
-                amount_in,
-                order_type,
-                expiration_time,
-                max_amount_out,
-            })),
-        ),
-        ExecuteMsg::WithdrawFilledLimitOrder { tranche_key } => Ok(Response::new().add_message(
-            Into::<CosmosMsg>::into(dex::MsgWithdrawFilledLimitOrder {
+        } => Ok(Response::new().add_message(MsgPlaceLimitOrder {
+            creator: env.contract.address.to_string(),
+            receiver,
+            token_in,
+            token_out,
+            tick_index_in_to_out,
+            limit_sell_price,
+            amount_in,
+            order_type,
+            expiration_time,
+            max_amount_out,
+            min_average_sell_price: "".to_string(), // TODO
```

Comment:
todo what? and by whom?

Reply:
forgot to check it.
removed the todo, since empty string is okay:
`// If min_average_sell_price is omitted limit_sell_price will be used instead`